airbyte-integrations/connectors/source-iterable/unit_tests/test_stream_events.py

```diff
@@ -4,199 +4,72 @@
 
 import json
 
+import pendulum
 import pytest
 import requests
 import responses
-from source_iterable.streams import Events
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.declarative.types import StreamSlice
+from source_iterable.source import SourceIterable
 
 
 @responses.activate
 @pytest.mark.parametrize(
     "response_objects,expected_objects,jsonl_body",
     [
         (
-            [
```

Comment:
Out of curiosity, why did you remove some of these events?

Reply:
Just typo.
Reverted.